django/forms/models.py

```diff
@@ -65,23 +65,20 @@ def construct_instance(form, instance, fields=None, exclude=None):
     return instance
 
 
-def save_instance(form, instance, fields=None, fail_message='saved',
-                  commit=True, exclude=None, construct=True):
+def save_instance(form, instance, fields=None, commit=True, exclude=None):
```

Comment:
Any objections to removing this method and simply inline the code in `ModelForm.save()`? As of b11564fd36587b1077bf7d77b62b7879cc08c382 it's no longer used elsewhere in Django and it isn't a documented public API, although I [found some usage on GitHub](https://github.com/search?p=1&q=save_instance+django&ref=reposearch&type=Code&utf8=%E2%9C%93).